composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "moxie-leean/wp-endpoints-archive",
+  "name": "moxie-lean/wp-endpoints-archive",
   "description": "wp-endpoints-archive",
```

Comment:
Correct my bad.

> On Mar 31, 2016, at 6:58 AM, Adam Fenton <notifications@github.com> wrote:
> 
> In composer.json <https://github.com/moxie-leean/wp-endpoints-collection/pull/2#discussion_r58048496>:
> 
> > @@ -1,5 +1,5 @@
> >  {
> > -  "name": "moxie-leean/wp-endpoints-archive",
> > +  "name": "moxie-lean/wp-endpoints-archive",
> >    "description": "wp-endpoints-archive",
> Shouldn't composer.json say collection instead of archive?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/wp-endpoints-collection/pull/2/files/86356df199c5bac0bd0cfbda93ff685a610a4205#r58048496>